### ISSUE TITLE
Log account and transaction

### ISSUE THREAD

**Issue by mikedotexe** [nearprotocol/near-shell#283]
@evgenykuzyakov said:
I want to make a request for near-shell. I think we should always log the transaction hash whenever shell makes any transaction. It will help us go to explorer and debug it in case something goes wrong

Other thoughts:
We should also start using account ID + transaction hash as a transaction identifier as otherwise it is tricky to get the result from the node

**Comment by ilblackdragon** [comment]
Agree, we should link to explorer for transaction issued and for transactions that create/change accounts we should link to them as well. 

It helps with checking status on tx, debugging, with promoting overall explorers as a tool that people should use.

Explorer should be part of configuration in `.neardev/config.json` to identify which explorer to use with something like this:
```
"explorer": {
   "mainnet": "https://explorer.nearprotocol.com",
   "betanet": "https://explorer.betanet.nearprotocol.com",
    ...
   "account_url": "/accounts/{account_id}",
   "tx_url": "/transactions/{tx_hash}",
}
```

**Comment by amgando** [comment]
duplicate https://github.com/near/near-shell/issues/268
